README.txt

```diff
@@ -14,9 +13,9 @@ doc [This folder contains the documentation for the project]
 
 include [This folder contains the header files]
 	message.hpp
-	receiverCadmium.hpp
-	senderCadmium.hpp
-	subnetCadmium.hpp
+	receiver_cadmium.hpp
+	sender_cadmium.hpp
+	subnet_cadmium.hpp
 
 src [This folder contains the c++ files used in the project]

```

Comment:
I would suggest to write it as ‘This folder contains source code files written in c++ for the project’. CPP files are called as source code files, hence suggesting to add it.

Reply:
Yes sure it sounds more specific and user can understand properly. I have made the changed you can have a look.

Reply:
sounds great and its perfect!